src/Components/Styles/Categories.css

```diff
@@ -0,0 +1,59 @@
+.categories {
+    margin: 1rem;
+    background-color: var(--white);
+}
+
+.categoryList {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+
+.categoryDiv {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
```

Comment:
Try using a utils since this a property you'll be using a lot

Reply:
Ok, I'll do that. Thanks for pointing out.